tests/unit/backend/wmg/test_query.py

```diff
@@ -96,8 +96,8 @@ def test__query_expression_summary_default_cube__returns_correct_results(self):
         with load_realistic_test_snapshot(TEST_SNAPSHOT) as snapshot:
             q = WmgQuery(snapshot)
             query_result = q.expression_summary_default(criteria)
-            query_sum = list(query_result[["sum", "nnz"]].sum())
-            expected = [804599.0, 370972.0]
+            query_sum = list(query_result[["sum", "nnz", "sqsum"]].sum())
+            expected = [804599.0, 370972.0, 1822108.0]
```

Comment:
this is the only test that i altered. there are existing tests that verify that the schema of the test snapshot matches the declared schema of the default cube. not sure what other tests would be valuable to add, since we now have coverage to:
- ensure the default cube has the `sqsum` array
- assertion on expected value of the sum of the `sqsum` array